test_conformance/api/test_api_consistency.cpp

```diff
@@ -116,8 +116,13 @@ int test_consistency_svm(cl_device_id deviceID, cl_context context,
         // needed here to ensure the SVM operations are complete before freeing
         // the SVM pointers.
 
-        clSVMFree(context, ptr0);
-        error = clEnqueueSVMFree(queue, 1, &ptr1, NULL, NULL, 0, NULL, NULL);
+        // These calls to free SVM purposefully passes a bogus pointer to the
+        // free function to better test that it they are a NOP when SVM is not
+        // supported.
+        cl_uint bogus = 0xDEADBEEF;
+        clSVMFree(context, &bogus);
+        void* pBogus = &bogus;
```

Comment:
Note that this will use a random stack address as the bogus pointer and not the pattern.

Reply:
Yeah, that was intentional: assigning a known bit pattern generates and MSVC error:

`warning C4312: 'type cast': conversion from 'unsigned int' to 'void *' of greater size`

Passing the stack pointer is a bit too clever though, so I've switched this to assign the known bit pattern instead.  I'll just live with the warning for now - there's no warning on with gcc.

(At some point I'd like to tidy up the MSVC warnings, but not now.)